### DISCUSSION TITLE
Inquiry about entropy calculation

### DISCUSSION THREAD

**Discussion by Chengeng-Yang** [Valdes-Tresanco-MS/gmx_MMPBSA#290] [General]
Hi there,

I was really excited when I found this project on github. I'm working on the calculation of protein conformational entropy change upon tensile stretch (that means, I don't have a ligand in this case). Do any of the following, Interaction Entropy calculations (https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/blob/master/examples/Entropy_calculations/Interaction_Entropy/README.md), NMODE Entropy calculations (https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/blob/master/examples/Entropy_calculations/nmode/README.md) or C2 Entropy calculations (https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/blob/master/examples/Entropy_calculations/C2_Entropy/README.md), apply to my case? Many thanks in advance.

**Comment by marioernestovaldes** [comment]
Unfortunately, no... gmx_MMPBSA methods are designed to work with complexes rather than individual molecules...

hope this helps!
